jsonnetfile.json

```diff
@@ -1,24 +1 @@
-{
```

Comment:
Removed this as these dependencies are not required till the dashboards are introduced. These are cloned with each build which is an overhead for now.